mm-mac2ipv4.bash

```diff
@@ -99,6 +99,7 @@ ip2=$(printf "%d" "0x$mac4")
 ip3=$(printf "%d" "0x$mac5")
 ip4=$(printf "%d" "0x$mac6")
 
-((ip2=ip2%64 + 64))
+((ip2=ip2 % 32 + 96))
```

Comment:
Second time I've seen this particular math. Is it worth extracting to a function?